addition.sh

```
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-x = a + b
+x = a + d
```

Shellcheck (0.10.0)

[error] Remove spaces around = to assign (or use [ ] to compare, or quote '=' if literal).

(SC2283)